CodyFire/Classes/WS/WS+Emit.swift

```diff
@@ -47,50 +47,20 @@ extension WS {
         return try jsonEncoder.encode(payload)
     }
     
-    private struct EM: Encodable {
-        let key: String
-    }
-    
     private struct GM: Encodable {
         let event: String
     }
     
-    public func emit<Model: WSEventModel>(data event: WSEventIdentifier<Model>, completion: (() -> Void)? = nil) {
```

Comment:
Please leave it `(data event` to be consistent with another method which is declared on line 32.